### ISSUE TITLE
no updates happening.. "Press any key to continue..."

### ISSUE THREAD

**Issue by przano** [grrttedwards/wow-addon-updater#207]
**Describe the bug**
This utility stopped working a couple months ago. Wrath classic. I've updated to latest version today 12/13/2022. This is what I get when running the windows.bat

Name           Prev. Version  New Version
────           ─────────────  ───────────
Press any key to continue . . .

**If you run `pipenv update` on the command line and try again, does the issue go away?**
No. Same result.

**Log output**
Please include the contents of `addon-updater.log`, from the root of the directory.
Either copy-paste the contents, or attach the file to this issue.

[addon-updater.log](https://github.com/grrttedwards/wow-addon-updater/files/10223069/addon-updater.log)

**Comment by afinkbeiner86** [comment]
The same is happening for me on retail.

**Log output**
[addon-updater.log](https://github.com/grrttedwards/wow-addon-updater/files/10227066/addon-updater.log)

**Comment by Stonga2** [comment]
Same happening for me as of yesterday.
[addon-updater.log](https://github.com/grrttedwards/wow-addon-updater/files/10227869/addon-updater.log)

**Comment by grrttedwards** [comment]
This is curse blocking your requests. There's nothing we can reliably do to fix this, sorry.

**Comment by afinkbeiner86** [comment]
I see, thank you for the reply!

**Comment by Stonga2** [comment]
Yeah it seems to work sometimes (like right now) and other times not. Must be that anti-bot thing you have mentioned earlier?

**Comment by grrttedwards** [comment]
Curse uses a service called CloudFlare in front before you get to talk to them, and sometimes cloudflare decides you are not worthy (or look like a bot). But I have no clue why it decides to break users of this tool, because it seems to work for long periods of time with no changes, and then suddenly stop working ... then it works again if we simply wait (usually)... 🫤

A user @omicron-o submitted a recent change that enables use of GitHub releases as an addon source. If you are able, I would try to switch as many addons over to a GitHub source or possibly another host.
